src/wallet/wallet.h

```diff
@@ -348,14 +348,13 @@ class CWalletTx : public CMerkleTx
     mutable CAmount nAvailableWatchCreditCached;
     mutable CAmount nChangeCached;
 
-    CWalletTx(const CWallet* pwalletIn, CTransactionRef arg) : CMerkleTx(std::move(arg))
+    CWalletTx(const CWallet* wallet, CTransactionRef arg) : CMerkleTx(std::move(arg)), pwallet(wallet)
```

Comment:
In commit "Avoid copying CWalletTx in LoadToWallet",

This and the `Init()` change could be in a different (trivial) commit.